### PR TITLE
Implemented aktualizr-info utility

### PR DESCRIPTION
**Pull request by patriotyk**

### CODE REVIEW

**Comment by cajun-rat** [comment]
The CI build failures appear to be due to the gcov setup:

~~~
/aktualizr/src/src/bootstrap.cc:126: undefined reference to `__gcov_init'
../libaktualizr_static_lib.a(bootstrap.cc.o):(.data+0x60): undefined reference to `__gcov_merge_add'
../libaktualizr_static_lib.a(crypto.cc.o): In function `_GLOBAL__sub_I_65535_0__ZNK9PublicKey10TypeStringB5cxx11Ev':
/aktualizr/src/src/crypto.cc:348: undefined reference to `__gcov_init'
../libaktualizr_static_lib.a(crypto.cc.o):(.data+0x60): undefined reference to `__gcov_merge_add'
../libaktualizr_static_lib.a(role.cc.o): In function `_GLOBAL__sub_I_65535_0__ZN6Uptane4RoleC2ERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE':
/aktualizr/src/src/uptane/role.cc:51: undefined reference to `__gcov_init'
../libaktualizr_static_lib.a(role.cc.o):(.data+0x60): undefined reference to `__gcov_merge_add'
collect2: error: ld returned 1 exit status
src/aktualizr_info/CMakeFiles/aktualizr-info.dir/build.make:112: recipe for target 'src/aktualizr_info/aktualizr-info' failed
~~~

**Comment by leon-anavi** [comment]
Hi @patriotyk, @cajun-rat,

Although building aktualizr from this revision and branch within meta-updater is fine, I recommend you merging the following fix: https://github.com/advancedtelematic/aktualizr/pull/412 

As far as I can see, building aktuakizr_info makes sense only if OSTree is enabled, right?

Best regards,
Leon

**Comment by patriotyk** [comment]
I just added the tests and build has passed. So now it is ready for merge

File: tests/conf_test_aktualizr_info.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+key_source = "pkcs11"
+public_key_path="01"
+private_key_path = "01"
+
+[p11]
+module = "/usr/lib/softhsm/libsofthsm2.so"
+pass = "123456"
+
+[provision]
+provision_path = "/home/patriotyk/Downloads/credentials(4).zip"
```
Comment (cajun-rat) [inline tests/conf_test_aktualizr_info.toml]: I think this is causing the following test failures

{code}

The following tests FAILED:
	 48 - aktualizr-info-images-root (Failed)
	 49 - aktualizr-info-director-root (Failed)
	 50 - aktualizr-info-director-target (Failed)
	 51 - aktualizr-info-images-target (Failed)
{code}

Comment (cajun-rat) [reply tests/conf_test_aktualizr_info.toml]: Weird, those work on travis but not on my machine

Comment (patriotyk) [reply tests/conf_test_aktualizr_info.toml]: I don't think so, but I have removed those lines. Could You run ctest -V -R aktualizr-info and give me output?

**Comment by cajun-rat** [comment]
I figured out that the aktualizr-info wasn't getting built ahead of the tests, added a fix

**Pull request merged**